nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java

```diff
@@ -1061,8 +1061,10 @@ public static boolean isValidNumericFilter(FilterMode mode) {
 		String alias() default "GridColumn";
 
 		/**
-		 * Apply styles to display value. This controls the display of value
+		 * <p>Apply styles to display value. This controls the display of value
 		 * with images and fonts
+		 * <p>When set to true, CSS classes added into <ul><li>{@link app_style.css}</li></ul> in will be added to the container element surrounding this component.
+		 * This can be used to apply additional styling, if necessary.
```

Comment:
What CSS classes will be added? How does the user of the framework know what styles they need to define in their custom stylesheets?

Consider adding a code example using `<pre>` tags that show an example of what a resulting HTML element might look like.

**Example Configuration**
```
@GridColumn(applyValueStyles = true)
private String status;
```

**Resulting HTML**
```
<pre>
<span class="status Active">
    ...
</span>
</pre>
```

**Example CSS**
```
.status.Active {
    ...
}
```